index.html

```diff
@@ -61,84 +61,116 @@ <h1 class="display-3">Bikes for Refugees</h1>
+								</a>
 							</div>
+							<aside class="article-title article-title--sidebar col-lg-8">
+								<a href="#">Bike dropoff event in Edinburgh on May 11</a>
+							</aside>
 						</article>
 						<article class="article row">
-							<a href="#">
-								<img class="img-fluid" src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/news-cycle.jpg">
-							</a>
-							<div class="article-title article-title--sidebar">
-								<a href="#">Help us cycle 4,797km</a>
+							<div class="col-lg-4">
+								<a href="#">
+									<img class="img-fluid" src="https://images.unsplash.com/photo-1444724834694-87238f9942f7?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=b93985d1d9b2367ef8d0f61d07640cf1&auto=format&fit=crop&w=800&q=60">
+								</a>
 							</div>
-							<p class="article-lead">We need you to help us cycle 4797km, the distance of Edinburgh to Damascus.</p>
+							<aside class="article-title article-title--sidebar col-lg-8">
+								<a href="#">Help us cycle 4,797km</a>
+								<p class="article-lead">We need you to help us cycle 4797km, the distance of Edinburgh to Damascus.</p>
+							</aside>
 						</article>
-							<a href="https://www.facebook.com/BikesforRefugeesScotland/events/?ref=page_internal" class="facebook-link">
+							<a href="#" class="facebook-link">
 								<span class="fa fa-facebook-square"></span>
 								All Facebook events
 							</a>
 						</div>
 					</div>
 				</div>
 			</div>
+
+			<div class="container">
+				<div class="row justify-content-between bike-more-info">
+					<div class="col-sm-12 col-lg-3 card align-self-center">
+						<h4 class="card-title">Check Availability</h4>
+					</div>
+					<div class="col-sm-12 col-lg-3 card">
+						<h4 class="card-title">Donate bikes</h4>
+					</div>
+					<div class="col-sm-12 col-lg-3 card">
+						<h4 class="card-title">Volunteer with us</h4>
+					</div>
+				</div>
+			</div>
+
+			<div class="container learn-more">
+				<div class="row">
+					<h2 class="heading-underline">Learn more</h2>
+					<div class="articles row justify-content-between">
+						<article class="article card col-sm-12 col-lg-5">
+							<div class="card-body">
+								<div class="article-title card-title">
+									Why do refugees need bikes?
+								</div>
+								<p class="article-lead card-text">Many refugees are placed in housing in areas where there are few jobs. Bikes provide low-cost transportation so that they can get to work, manage child care and do their shopping.</p>
+								<p class="article-read-more">
+									<a href="#">
+										<span class="fa fa-caret-right"></span>
+										Learn more
+									</a>
+								</p>
+							</div>
+						</article>
+
+						<article class="article card col-sm-12 col-lg-5">
```

Comment:
You can't have horizontal padding/margins on custom classes where you also use `col-...` classes, it undoes what the grid does! 🙂 
To avoid the boxes touching, move the border/background color one level down (or move the `col-` class one level up. You will have to add a `div` or similar...